clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -683,21 +686,55 @@ public Node provide() {
         }
     }
 
+    class RetryContext {
+
+        private int tries = 0;
+        private long nextAllowedTryMs = 0;
+
+        public int getTries() {
+            return tries;
+        }
+
+        public long getNextAllowedTryMs() {
```

Comment:
nit: Normally for getters we have the convention of dropping the `get` from the method name.

Reply:
Refactored